srcCs/perftest_publisher.cs

```diff
@@ -1267,6 +1257,118 @@ bool ParseConfig(string[] argv)
             return true;
         }
 
+        private void PrintConfiguration() {
+
+            StringBuilder sb = new StringBuilder();
+
+            sb.Append("\nPerftest Configuration:\n");
+
+            // Throughput/Latency mode
+            if (_IsPub) {
```

Comment:
the { in the next file

Reply:
Something that confused me in the code standard of C# is "{" in a condition in the same line or the next. Fix it in following the standard of C#/

Reply:
We need to come up with a standard for this. I agree. Lets discuss it in the next Performance meeting.